R/FilterStateRange.R

```diff
@@ -141,8 +140,7 @@ RangeFilterState <- R6::R6Class( # nolint
                           ...) {
       checkmate::assert_numeric(x, all.missing = FALSE)
       if (!any(is.finite(x))) stop("\"x\" contains no finite values")
-
-      # validation on x_reactive here
+      checkmate::assert_class(x_reactive, 'reactive')
 
       do.call(
```

Comment:
`super$initialize` is still in the beginning of the constructor body. It doesn't correspond with #222 where super class sets `keep_inf` and I see this to be potential error as `private$keep_inf` is prepared too late (see the line 164 below). I guess you need to synchronize your actions with @chlebowa

Reply:
`RangeFilterState` was not pushed yet, working on it now.